tests/unit/test_step.py

```diff
@@ -70,17 +70,29 @@ def test_implemented_step(implemented_step_params) -> None:
     assert step.output_slots == {"step_1_main_output": OutputSlot("step_1_main_output")}
 
 
-def test_implemented_step_update_implementation_graph(
-    implemented_step_params, default_config: Config
+def test_basic_step_update_implementation_graph(
+    basic_step_params: Dict[str, Any], default_config: Config
 ) -> None:
-    step = BasicStep(**implemented_step_params)
+    step = BasicStep(**basic_step_params)
     subgraph = nx.MultiDiGraph()
     subgraph.add_node(step.name, step=step)
     step.update_implementation_graph(subgraph, default_config["pipeline"][step.name])
     assert list(subgraph.nodes) == ["step_1_python_pandas"]
     assert list(subgraph.edges) == []
 
 
+def test_basic_step_get_implementation_node_name(
```

Comment:
It seems like this should be beefed up a bit to make sure it handles a looped step correctly?

Reply:
My logic here was that `get_implementation_node_name` only generically uses the step_name and name of its parent step, not any feature unique to loop steps--the only reason I'm using BasicStep as a parent is because I can't instantiate an abstract Step. We cover the loop step case (though not *exclusively* the behavior of `get_implementation_node_name`) by checking for the correct node names in `test_loop_update_implementation_graph`. What coverage would you say is missing from this?